### DISCUSSION TITLE
STM32H573xx Port

### DISCUSSION THREAD

**Discussion by bonsai-eater** [micropython/micropython#13787] [STM32 / Pyboard] [answered]
Hello, there!

I need to use Micropython for my STM32H573I-DK and i see that there is no micropython firmware for this microcontroller. Is it possible to recieive some guidance or tips on where i should start if i want to make a port for it myself?

**Comment by robert-hh** [comment]
There is a board definition for that board, and the firmware can be built. So you could build it yourself.I do not know why it is not on the download pages. Attached is a copy of a fresh built firmware.dfu file.

[firmware.zip](https://github.com/micropython/micropython/files/14452580/firmware.zip)

**Reply by robert-hh** [reply, marked as answer]
For building it yourself, install the files and tools as written in the README of Micropython and of the stm32 port, and then call:

make BOARD=STM32H573I_DK

**Reply by bonsai-eater** [reply]
Thank you, i have built the firmware myself with provided instructions and it works well.
